index.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <!-- Global site tag (gtag.js) - Google Analytics -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=G-WK01NEHZ12"></script>
+        <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){window.dataLayer.push(arguments);}
+        gtag('js', new Date());
+
+        gtag('config', 'G-WK01NEHZ12');
+        </script>
+        <meta charset="utf-8">
+        <title> Itumeleng </title>
+        <link rel="stylesheet" type="text/css" href="styles/template2.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;1,100;1,600&display=swap" rel="stylesheet">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+      <section class="home-page">
+        <header>
+          <nav id="nav">
+            <a class="logo" href="#">My Logo</a>
+
+            <nav class="hamburger"><img src="statics/images/Union.png" alt="Menu"></nav>
+            <ul class="internal-link">
+               <li><a href="#my_work">Portfolio</a></li>
```

Comment:
- [ ] Make sure the hamburger menu icons display works correctly. Please fix :wrench: